__tests__/app.test.js

```diff
@@ -520,6 +520,43 @@ describe("app", () => {
     });
   });
 
+  describe("GET /users", () => {
+    test("Responds with a status of 200 for the right request.", () => {
+      return request(app).get("/api/users").expect(200);
+    });
+    test("200 - Responds with an array of users to the client.", () => {
+      return request(app)
+        .get("/api/users")
+        .expect(200)
+        .then(({ body }) => {
+          
+          const users = body.users;
+          
+          expect(Object.keys(users[0]).length).toBe(3);
```

Comment:
You may want to remove this, just in case in the future we ever want to expand this table to include more keys. This would make this test quite brittle.

Reply:
Thank you for the feedback, done :)